x/ecocredit/migrations/v3/patch.go

```diff
@@ -45,12 +45,21 @@ func patchMainnet(ctx context.Context, ss api.StateStore, oldBatchDenomToNewDeno
 	//  C01-20190101-20191231-002  -  "2022-05-06T01:33:19Z"
 	//  C01-20150101-20151231-003  -  "2022-05-06T01:33:25Z"
 	//  C01-20150101-20151231-004  -  "2022-05-06T01:33:31Z"
+	//  C01-20150101-20151231-005  -  "2022-06-17T00:04:31Z"
```

Comment:
It would be nice to cross-check batch issuance date.
```
Old denom =  C01-20190101-20191231-001  new denom =  C01-002-20190101-20191231-001   issuance date =  2022-05-06T01:33:13Z
Old denom =  C01-20190101-20191231-002  new denom =  C01-002-20190101-20191231-002   issuance date =  2022-05-06T01:33:19Z
Old denom =  C01-20150101-20151231-003  new denom =  C01-001-20150101-20151231-001   issuance date =  2022-05-06T01:33:25Z
Old denom =  C01-20150101-20151231-004  new denom =  C01-001-20150101-20151231-002   issuance date =  2022-05-06T01:33:31Z
Old denom =  C01-20150101-20151231-005  new denom =  C01-001-20150101-20151231-003   issuance date =  2022-06-17T00:04:31Z
Old denom =  C01-20190101-20191231-006  new denom =  C01-002-20190101-20191231-003   issuance date =  2022-06-17T00:04:37Z
Old denom =  C01-20150101-20151231-008  new denom =  C01-001-20150101-20151231-005   issuance date =  2022-06-17T00:04:51Z
Old denom =  C01-20150101-20151231-007  new denom =  C01-001-20150101-20151231-004   issuance date =  2022-06-17T00:04:43Z
```

Reply:
Double checked https://hackmd.io/20SJJwvYRde6YC-jRN5Pdg